showcase/404.html

```diff
@@ -0,0 +1,22 @@
+<html>
+<head>
+	<script>
+		sessionStorage.redirect = location.href;
```

Comment:
@Olivia, I don't think we need the "404" page in the Showcase. This should be present only at the root of the _gh-pages_ branch... so we only need this one: https://github.com/NationalBankBelgium/stark/blob/gh-pages/404.html

Reply:
yup, makes sense actually --' thanks!